lumicks/pylake/force_calibration/power_spectrum_calibration.py

```diff
@@ -78,6 +78,26 @@ def plot(self):
         self.ps_model.plot(label="Model")
         plt.legend()
 
+    def plot_residual(self):
```

Comment:
nitpick: Could this be `plot_spectrum_residuals(self)` instead? Would save extra googling what residuals we're talking about 😅